content/authors/Meenakshi-Siddharthan/_index.md

```diff
@@ -6,15 +6,16 @@ bio:
 
 education:
   courses:
-  - course: TBC
-    institution: TBC
-    year: TBC
-  - course: TBC
-    year: TBC
+  - course: MSc Clinical Neuroscience
+    institution: University College London
+    year: 2018
+  - course: BSc Applied Psychology
+    institution: Durham University
+    year: 2017
 
 
 email: "m.siddharthan@ntu.edu.sg"
-#interests:
+#interests: Lifelong learning, Cognition, Brain stimulation
```

Comment:
I think you need to remove the "#" otherwise it is considered a comment if I'm not mistaken right @Tam-Pham ?

Reply:
Yes! That's right! @MeenaSiddharthan

Reply:
And you can have the interests as a list (can take a look at other profiles for reference ^^)